tcms/testplans/views.py

```diff
@@ -393,7 +393,6 @@ def post(self, request, plan_id):
 
         return JsonResponse({'rc': 0, 'response': 'ok'})
 
-
```

Comment:
this breaks flake8. Run tests locally before you push and checkout Travis CI statuses after you do.